### PR TITLE
[zh] sync release news 1.6.x

### PR DESCRIPTION
**Pull request by SAMZONG**
Signed-off-by: samzong.lu <samzong.lu@gmail.com>

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.6.x/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: 1.6.x 版本
```
Comment (hanxiaop) [inline content/zh/news/releases/1.6.x/_index.md:2]: Normally, we don't maintain things that have reached EOL. Please see the supported release page at https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases to see the list of versions we are maintaining

Comment (SAMZONG) [reply content/zh/news/releases/1.6.x/_index.md:2]: 😭 Unfortunately, At before, I didn't notice the information here.

**Comment by windsonsea** [comment]
1.6.x is too old -_!

**Comment by SAMZONG** [comment]
Oops... 😂

**Pull request closed without merging**